src/plugins/links/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -137,7 +137,7 @@ export const DashboardLinkComponent = ({
 
   return loadingDestinationDashboard ? (
     <li id={`${id}--loading`}>
-      <EuiButtonEmpty size="s" isLoading={true} data-test-subj={`${id}--loading`}>
+      <EuiButtonEmpty size="s" isLoading={true} data-loading data-test-subj={`${id}--loading`}>
```

Comment:
Hmmm... do we maybe want to make use of the `RenderCompleteDispatcher` instead? 🤔 This is how Lens, the image embeddable, and saved searches handle this... But we seem to be pretty inconsistent. @ThomThomson any thoughts here? I know we've discussed this before for controls, and we did this "cheap and nasty" way there 😆

Reply:
Yeah it looks like it should be pretty straightforward to use the `RenderCompleteDispatcher` via the abstract Embeddable. I'm not completely sure how that ends up setting the attribute, but we definitely _should_ be using it. The more consistent we are, the easier it will be to update it all.